src/commands/main.mts

```diff
@@ -212,6 +212,7 @@ export const createMainCommand = () => {
   createSwitchCommand(program)
   createUnlinkCommand(program)
   createWatchCommand(program)
+  createLogsCommand(program)
```

Comment:
ohhh right these show up in order in the `netlify help` command. Can you alphabetize it pls?

Reply:
you're joking

Reply:
nah I'm gonna refactor the help command